src/server/config.rs

```diff
@@ -128,6 +129,9 @@ pub struct Config {
 impl Default for Config {
     fn default() -> Config {
         let cpu_num = SysQuota::new().cpu_cores_quota();
+        let mut background_thread_count = (cpu_num.round() as usize + 7) / 8;
```

Comment:
Please add it into `etc/config-template`.

Reply:
Ok.